packages/core/src/graph/graph.ts

```diff
@@ -2130,16 +2159,32 @@ export default abstract class AbstractGraph extends EventEmitter implements IAbs
     const nodes: NodeConfig[] = [];
     const edges: EdgeConfig[] = [];
     const combos: ComboConfig[] = [];
+
+    const getModelWithInitialStates = (item: INode | IEdge | ICombo): NodeConfig | EdgeConfig | ComboConfig => {
+      const model = item.getModel();
+      model.initialStates = {};
```

Comment:
May be we could call it `.states` rather than `.initialStates` because it makes more sense when saving the graph.

Reply:
-___-